backend/lambda/function.mjs

```diff
@@ -39,66 +49,114 @@ export const handler = async (event, context) => {
     try {
         switch (event.requestContext.routeKey) {
             case "POST /levels":
```

Comment:
Quick note:

`event.requestContext.routeKey` should be `event.requestContext.resourceId`.

I tried running the lambda yesterday and it wasnt working until I made that change :)